parts/linux/cloud-init/artifacts/ubuntu/mellanox_mlnx_ofed.list

```diff
@@ -0,0 +1,9 @@
+#
+# Mellanox Technologies Ltd. public repository configuration file.
+# For more information, refer to http://linux.mellanox.com
+#
+# For future reference:
+# https://network.nvidia.com/support/mlnx-ofed-public-repository/
+# https://www.mellanox.com/downloads/ofed/RPM-GPG-KEY-Mellanox
+# [mlnx_ofed_24.10-1.1.4.0_base]
```

Comment:
how often will this version change ? 1.1.4.0 ?

Reply:
before MAI request next feature require another driver upgrade